inline.go

```diff
@@ -6,6 +6,7 @@ import (
 	"bytes"
 	"regexp"
 	"strconv"
+	"strings"
```

Comment:
again, not against cleanup (very much welcome :) ), but why all the changes in this file as well?